frontend/src/styles/App.css

```diff
@@ -2,3 +2,30 @@
   width: 100px;
   height: 100px;
 }
+
+/* Style for the hamburger button */
```

Comment:
We don't have to define any style in a global style file instead we can create separate style files for the specific component.

Also, I would suggest to use the` <ActionIcon />` component of mantine, this allow use to use icon as button.
https://mantine.dev/core/action-icon/

Example:
`
<ActionIcon onClick={open}>
                <IconMenu2 />
              </ActionIcon>
              `